test/modules/CloudToDeviceMessageTester/CloudToDeviceMessageTesterMetadata.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace CloudToDeviceMessageTester
+{
+    using System;
+    using Microsoft.Azure.Devices.Client;
+
+    public class CloudToDeviceMessageTesterMetadata
```

Comment:
nit: no need to have this hierarchy.  Just rename these structs to "C2DTestSharedSettings", "C2DTestSenderSettings", and "C2DTestReceiverSettings".